chatterbot/conversation/conversation.py

```diff
@@ -0,0 +1,20 @@
+import uuid
+from chatterbot.queues import ResponseQueue
```

Comment:
`ResponseQueue` will need to be removed and the methods are going to be updated to use database queries.